src/probe/probe.c

```diff
@@ -132,7 +132,6 @@ static int probe_dma_buffer_init(struct probe_dma_buf *buffer, uint32_t size,
  */
```

Comment:
The git commit prefixes are not quite correct. SOF follows same style as in Linux kernel and git commit summary line should have a prefix that matches the code that is changed. E.g. for this commit, this should be "probe: local dma_sg_config is no more needed"

Similarly for other commits. Please see "git log src/path/to/file.c" for examples from older commits.